godtools/ViewControllers/Platform/GTLanguageSettingsViewController.xib

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I know you're still working on this file, but make sure you check it on different device sizes. It works well in 6 and 6 Plus, but the layout overlaps on 5S and 4S devices.

![simulator screen shot apr 20 2017 10 39 38 am](https://cloud.githubusercontent.com/assets/3768029/25236457/e25a7fe8-25b5-11e7-835e-af59338cc489.png)